apps/heft/src/metrics/MetricsCollector.ts

```diff
@@ -98,9 +103,14 @@ export class MetricsCollector {
    * Record metrics to the installed plugin(s).
    *
    * @param command - Describe the user command, e.g. `start` or `build`
-   * @param params - Optional parameters
+   * @param parameterMap - Optional map of parameters to their values
+   * @param performanceData - Optional performance data
    */
-  public record(command: string, performanceData?: Partial<IPerformanceData>): void {
+  public record(
+    command: string,
+    parameterMap?: Record<string, string>,
```

Comment:
The `parameterMap` name is a little misleading since this takes a `Record`, not a `Map`